extensions/src/ai_storybook_dancing/index.ts

```diff
@@ -5,14 +5,14 @@ import { announce, requestDanceMove, untilMessageReceived } from "./messaging";
 
 const details: ExtensionMenuDisplayDetails = { name: "Dancing Activity for AI Storybook" };
 
-const dance = async (move: "hop") => {
+const dance = async (move: string = "hop") => {
```

Comment:
Instead of setting the argument type to `string` (which is of course is a huge category that could include lots of strings), it'd be best to restrict it to the specific strings we suspect. That's what the `"hop"` argument type was doing before. 

So you can either do this inline within the argument: 
```ts
const dance = async (move: "hop" | "swivel left" | "swivel right") => {
```
or you can define a new type and use that instead:
```
type DanceMove = "hop" | "swivel left" | "swivel right";

const dance = async (move: DanceMove) => {
```